public/views/index-template.html

```diff
@@ -180,7 +180,7 @@
           <div class="preloader__logo"></div>
         </div>
       </div>
-      <div class="preloader__text">Loading Grafana</div>
+      <div class="preloader__text">Loading Insights</div>
       <div class="preloader__text preloader__text--fail">
         <p>
           <strong>If you're seeing this Grafana has failed to load its application files</strong>
```

Comment:
`<strong>If you're seeing this Grafana has failed to load its application files</strong>`
I think this Grafana text replacement is missed with Insights.

Reply:
Should we worry about half checks failed sir?

Reply:
Those are the grafana jobs which may not work for us in this fork